### ISSUE TITLE
Unable to locate package libre2-dev                                      

### ISSUE THREAD

**Issue by flip111** [scrapinghub/portia#426]
When running

```
sudo ./provision.sh install_splash install_python_deps
```

the process exit with an error

```
Hit http://ports.ubuntu.com trusty/universe Translation-en
Reading package lists...
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package libre2-dev
Command failed (exitcode: 100), exiting...
```

After that i installed https://github.com/google/re2 from source, but i still get the same error. My OS is ubuntu 14.04. What to do?

`Linux pi 3.18.0-14-rpi2 #15-Ubuntu SMP PREEMPT Wed Feb 18 10:43:15 UTC 2015 armv7l armv7l armv7l GNU/Linux`

**Comment by ashlesh688** [comment]
Hi I am facing the same error as above can you please share the solution with me

**Comment by METAJIJI** [comment]
Seems need add `universe` repo http://security.ubuntu.com/ubuntu/pool/universe/r/re2/
But `libre2-dev` available from ubuntu `xenial` 16.04, not `trusty` 14.04 more info: https://pkgs.org/download/libre2-dev
